integration-tests/testsetups/vrfv2.go

```diff
@@ -126,7 +126,7 @@ func (v *VRFV2SoakTest) Run(t *testing.T) {
 	//todo - need to find better way for this
 	time.Sleep(1 * time.Minute)
 
-	loadTestMetrics, err := v.Inputs.ConsumerContract.GetLoadTestMetrics(nil)
+	loadTestMetrics, err := v.Inputs.ConsumerContract.GetLoadTestMetrics(context.Background())
```

Comment:
It looks like there is already a context in scope here:
```suggestion
	loadTestMetrics, err := v.Inputs.ConsumerContract.GetLoadTestMetrics(testContext)
```

Reply:
Looks like that context already has Done called on it.

Reply:
What do you mean? `Done` just reports the state. Is this code supposed to execute after the loop has exited? That other context should be wired to the test instead of background anyways, so this could just take the parent of the previous one.